lib/web/apiserver.go

```diff
@@ -459,7 +459,7 @@ func (h *Handler) bindDefaultEndpoints(challengeLimiter *limiter.RateLimiter) {
 
 	// Unauthenticated access to retrieving the script used to install
 	// Teleport
-	h.GET("/webapi/scripts/installer", httplib.MakeHandler(h.installer))
+	h.GET("/webapi/scripts/:installer", httplib.MakeHandler(h.installer))
```

Comment:
Would `webapi/scripts/installer/:name` make more sense? What if we need other scripts that are not installers?

Spoiler alert: we are planning on other scripts :-)